hack/istio/multicluster/kiali-prepare-remote-cluster.sh

```diff
@@ -14,7 +14,7 @@ set -u
 ##############################################################################
 
 # Used to name all the resources on the remote cluster.
-KIALI_RESOURCE_NAME="kiali-remote-access"
+KIALI_RESOURCE_NAME="kiali"
```

Comment:
you sure you want to rename this? IIRC it was specifically not named "kiali" to avoid potential confusion with a real kiali install (say from the helm chart or operator).

Reply:
Eh no I probably edited this while testing out changes and forgot to remove it. Good catch.

Reply:
Note also - this is an customer-facing script that people in the community are using now. Is this going to cause a problem if we change the name on them?